data/human/human missions.txt

```diff
@@ -6893,3 +6895,387 @@ mission "Lost Dog: Homeward Bound"
+				die
+
+			label "see us"
+			`	"I have no idea," says Bunker. "But it's probably time to go."`
+			choice
+				`	"Call back the drones immediately."`
+					goto "call drones"
+				`	"I think we have to leave the drones behind."`
+
+			label "no can do"
+			`	Bunker hits the recall button for the drones anyway. "This display is a low resolution proxy," explains Bunker. "If we leave the drones, we'll lose all the footage we need. It looks like the Quicksilvers aren't coming straight for us. We have time. The drones are less than two minutes out."`
+			choice
+				`	"We're leaving. Now!"`
+				`	"Fine. Get to the cargo bay and lock down the drones as soon as they land. We take off on your mark."`
+					goto escape
+
+			`	"I can't let you do that, <first>," says Bunker, standing between you and the controls. "The drones are almost here. Just let me go to the cargo bay to lock them down and we're out of here.`
+			choice
+				`	(Force him to stand down.)`
+					goto blaster
+				`	(Let him go to the cargo bay and lock down the drones.)`
+					goto escape
+
+			label "call drones"
+			`	"Aye aye, Captain," he says as he hits a button. "Drone ETA: two minutes, fifteen seconds."`
+			`	"Get to the cargo bay," you say. "Lock down the drones as soon as they arrive and get belted in. We'll take off on your mark."`
+
+			label escape
+			`	Less than a minute later, you hear Bunker on the comms. "Drones are locked down and ready to go."`
+			`	You immediately raise the shields and take off. The moment you pull out of the canyon, pirate Quicksilvers appear from every direction. You shunt the <ship>'s repulsors into full blast and accelerate into space with the pirates in hot pursuit.`
+				launch
+
+			label blaster
+			`	You get up and point your pistol at Bunker while ordering him to stand down. "Don't do this," he says, shrinking from you as he puts his hands up. As you are restraining him, you notice the humming of thrusters from outside - then the <ship> receives a hail of particle cannon fire out of nowhere! You turn around and see pirate Quicksilvers all converging towards you, tearing through the <ship>'s hull. You shunt the <ship>'s repulsors to full blast to get into orbit, but a volley of particle fire breaks through and takes the repulsors offline. The last thing you experience is the ship slowing to a halt, pausing in the air for just a moment before the pilot seat goes out from under you, the cockpit filled with screams as the <ship> makes a one way trip towards the surface.`
+				die
+
+	npc
+		personality heroic nemesis vindictive launching marked
```

Comment:
Do we really want it to be the case that the Syndicate wouldn't fire on these ships, though? That would imply that the whole of the Syndicate knows that these are pirates that they are making deals with. Would that not make it more obvious that the Syndicate is involved in the slave trade? That would also presuppose that the Syndicate actually are involved in the slave trade, rather than the possibility that the sweatshop is a single rogue element of some sort.

Reply:
The Syndicate is supposed to have enough plausible deniability to consciously make the decision to employ slaves – I believe that's what Lake told me back in November, anyway.

Reply:
Got rid of `marked` but still waiting on your opinion of the above.

Reply:
While the Syndicate are undoubtedly not the most benevolent force in the galaxy, I don't think it makes sense to continually display them as malevolent at the same time, as that becomes very one-note/dimensional.

A dynamic I've described in the past is that I'd like to see the Syndicate as being portrayed as law-abiding... except for the fact that their level of control in Parliament allows them to influence the law. This means that the Syndicate's "plausible deniability" is the fact that they actually do follow the laws of the Republic, even if they are the ones shaping those laws to a major extent. Given that, the Syndicate as an organization wouldn't knowingly disregard slavery laws within their own boundaries, as we know that the Republic outlaws slavery. At worst, the Syndicate would make deals with pirates/warlords who make use of slave labor on pirate planets, but even those would be special circumstances/something that only a few in the Syndicate would know of. Your average Syndicated Security captain isn't going to have orders to not fire on a pirate ship that's chasing a merchant captain through Syndicate space because "Oh wait, that's one we're making deals with."

Reply:
Alright, that sounds fair. Perhaps these sweatshops are run by the managers who are ruthless in trying to meet production numbers and need to pad their manufacturing lines to get that edge. I'll take a look at what the original author of the PR wrote for the next part and see how they showed the exhibition, since I think that is described. A convo that pops up saying that the Syndicate has dropped another "trusted provider" shouldn't be too hard though.